README.md

```diff
@@ -231,6 +231,25 @@ In the example above, `push_periodic_task` is used to save the specified task to
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create new Pull Request
 
+### Running tests locally
```

Comment:
can you please add a couple of notes to the README about the worker pool shutdown?

Reply:
Added a reference to it in README, and also updated the simple_worker example to use signal-hook + shutdown. I think having that in the simple_worker makes sense, as apposed to having a separate example with it.